R/cleanup.R

```diff
@@ -713,37 +701,37 @@ cleanGeneDescription <- function(prot, column) {
 #' selectLongestDuplicate()
 #' }
 selectLongestDuplicate <- function(prot, column) {
-    col <- sym(column)
-
-    prot$row.orig <- 1:nrow(prot)
-
+    col <- rlang::sym(column)
+    prot <- prot %>% 
+        mutate(row.orig = seq_len(n()))
```

Comment:
Marking Original Rows:

mutate(row.orig = seq_len(n())) is used to create a unique identifier for each row, ensuring original row positions are preserved even after filtering. This addition is necessary for accurately managing duplicate entries.